octodns/provider/ns1.py

```diff
@@ -69,10 +69,13 @@ def _data_for_CAA(self, _type, record):
         }
 
     def _data_for_CNAME(self, _type, record):
+        value = None
+        if len(record['short_answers']) == 1:
+            value = record['short_answers'][0]
```

Comment:
Slightly more pythonic way of writing that would be using try/except, thought that's starting to get into preference territory. 

```python
try:
    value = record['short_answers'][0]
except IndexError:
    value = None
```